geniza/corpus/management/commands/import_data.py

```diff
@@ -517,8 +554,8 @@ def get_edit_history(self, input_by, date_entered, pgpid=None):
         """
 
         # split both fields by semicolon delimiter & remove whitespace
-        all_input = [i.strip() for i in input_by.split(";")]
-        all_dates = [d.strip() for d in date_entered.split(";")]
+        all_input = [i.strip() for i in input_by.split(";") if i]
+        all_dates = [d.strip() for d in date_entered.split(";") if d]
```

Comment:
Is this right? I thought we needed the two fields to match up (to the degree possible)

Reply:
The demerge spreadsheet has `' '` in these columns, which threw an error. I realize that this quick fix may not be sufficient. This might be worth talking over.

Reply:
defer to @thatbudakguy on this one!

Reply:
i think this is ok; we pad the lists to the same length later. also we might want to just duplicate/fill down the data in the demerge spreadsheet rather than leaving in the `' '`, which would eliminate this problem